app/components/app_vaccinate_form_component.rb

```diff
@@ -36,6 +36,10 @@ def campaign_name
   end
 
   def vaccination_initial_delivery_sites
-    %w[left_arm right_arm other]
+    [
+      OpenStruct.new(value: "left_arm", label: "Left arm (upper position)"),
```

Comment:
In `VaccinationRecord` we have `left_arm` and `left_arm_upper_position` as values for the enum `delivery_site`. Should we be using the latter here?

Reply:
You're right!